src/services/auth/Login.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { withRouter } from 'react-router';
+import ssoProviders from 'settings/ssoProviders';
```

Comment:
this needs to be imported directly from `settings` so the user can over-write it in their theme